src/index.html

```diff
@@ -20,19 +20,12 @@ <h1 class="heading">Leaderboard</h1>
                 <h2>Recent Scores</h2>
                 <button class="refresh-btn">Refresh</button>
             </div>
-            <ul class="table-scores">
-                <li class="score">Name: 100</li>
-                <li class="score">Name: 20</li>
-                <li class="score">Name: 52</li>
-                <li class="score">Name: 78</li>
-                <li class="score">Name: 96</li>
-                <li class="score">Name: 59</li>
-            </ul>
+            <ul class="table-scores"></ul>
         </div>
         <div class="submit-score">
-            <form>
-                <input type="text" placeholder="Your Name" class="name-input">
-                <input type="text" placeholder="Your Score" class="score-input">
+            <form class="submit-game">
+                <input type="text" placeholder="Your Name" class="name-input" required>
+                <input type="number" placeholder="Your Score" class="score-input" required>
```

Comment:
- [ ] [Optional] you can consider add some kind of validation so when a user enters a long score, the UI will not break

![Screenshot_20220521_194126](https://user-images.githubusercontent.com/25227022/169665276-efbc0ecd-25e1-45c2-8fdc-3c57b2ca8c7f.png)

Reply:
Done ✔
@Ismailco however, I had a question, how can I delete this number have already broken the UI?